doc/tutorials/bare_flame_game.md

```diff
@@ -108,17 +106,16 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  flame: ^1.1.0
+  flame: ^--VERSION--
 ```
 
-After that, press the `[Pub get]` button at the top of the window; or
-alternatively you could run command `flutter pub get` from the terminal. This
-will "apply" the changes in `pubspec` file to your project, in particular it
-will download the Flame library which we have declared as a dependency. In the
-future, you should run `flutter pub get` whenever you make changes to this
-file.
+After that, press the `[Pub get]` button at the top of the window, or you could
+run command `flutter pub get` from the terminal. This will "apply" the changes
+in `pubspec` file to your project, in particular, it will download the Flame
+library which we have declared as a dependency. In the future, you should run
+`flutter pub get` whenever you make changes to this file.
```

Comment:
This had the wrong line length before, but could be fixed in this PR. :)

Reply:
Fixed.